Objects/object.c

```diff
@@ -1638,6 +1638,11 @@ none_bool(PyObject *v)
     return 0;
 }
 
+static Py_hash_t none_hash(PyObject *v)
+{
+    return 0xBADCAB1E;
```

Comment:
It's not an error, no certainly doesn't involve cables.  ;-)
```suggestion
    return 0xFCA86420;
```